wiki/Beatmap_Discussion/en.md

```diff
@@ -82,9 +82,11 @@ Hype train is similar to the old star priority system. A user can submit Praise
 
 *Note: the creator can't Hype up their own beatmap. Praise must be done in the General (All Difficulties) section to add Hype. It will not work through the General or the Timeline tabs.*
 
+Hype must be done in the General (All Difficulties) section to add Hype. It will not work through the General or the Timeline tabs. Players are required to play through a difficulty in the beatmapset in order to add Hype to a Beatmap.
```

Comment:
Hype must be done in the General (All Difficulties) section to add Hype -> If you want to add Hype, it must be done in the General (All Difficulties) section.

"must be done in the General section" "it wont work through the General tab" what the fuck
why is a "beatmap" capitalized at the end of the line

Reply:
> Hype must be done in the General (All Difficulties) section to add Hype -> If you want to add Hype, it must be done in the General (All Difficulties) section.
OK

---
>"must be done in the General section" "it wont work through the General tab" what the fuck
why is a "beatmap" capitalized at the end of the line
what, can you explain what you suggested?

Reply:
Those sentences exclude themselves.
The beatmap shouldnt be capitalized as it is on the end of the line.

Reply:
sorry, I wanted clarification for the things in quotes, not the caps of beatmaps...
was that a suggestion or just pointing something out with no suggestion?

Reply:
disregard-- I found out what you had meant and combined the sentences together.